umbraco/Umbraco.CMS.Backoffice
# Tiptap extension: Embedded Media

## Description

Adds the Embedded Media feature to the Tiptap toolbar, opening the modal to enter a URL and viewing the OEmbed preview.

## Types of changes


## How to test?

Press the Embed button in the toolbar to open the Embedded Media modal, enter a URL, e.g. "`https://www.youtube.com/watch?v=J---aiyznGQ`", press the Retrieve button, review the other options, then press Submit, the embedded media will appear in the editor.

To update the embedded media, select the item (it should be highlighted) then press the Embed button in the toolbar, the modal will open with the OEmbed options, update and submit, the editor should update accordingly.
